src/jobs/validators-update/validators-update.service.ts

```diff
@@ -57,11 +56,16 @@ export class ValidatorsUpdateService {
   }
 
   private updateMetrics() {
-    // TODO: IF
-    this.lastBlockTimestamp &&
+    if (this.lastBlockTimestamp) {
       this.prometheusService.validatorsRegistryLastTimestampUpdate.set(this.lastBlockTimestamp);
-    this.lastBlockNumber && this.prometheusService.validatorsRegistryLastBlockNumber.set(this.lastBlockNumber);
-    this.lastSlot && this.prometheusService.validatorsRegistryLastSlot.set(this.lastSlot);
+    }
+
+    if (this.lastBlockNumber) {
+      this.prometheusService.validatorsRegistryLastBlockNumber.set(this.lastBlockNumber);
```

Comment:
since we have two types of blocks: EL and CL maybe it's better to have separate names, like: clBlockNumber, elBlockNumber